### PR TITLE
LP-2493 Handle pre reqs, when each course has two versions

### PR DESCRIPTION
**Pull request by zahid-arbisoft**
**[LP-2493](https://philanthropyu.atlassian.net/browse/LP-2493)** Handle prereqs, when each course has two versions (English & Arabic)

@ahmed-faraz46 @asadali145 This code is already reviewed in another [PR](https://github.com/OmnipreneurshipAcademy/edx-platform/pull/130), which was created for old (juniper) develop branch.

### CODE REVIEW

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/945/workflows/workflows_id/jobs/945/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/course_meta/admin.py (100%)
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/management/commands/tests/test_update_is_prerequisite_courses_passed.py (100%)
openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py (97.5%): Missing lines 33
openedx/adg/lms/applications/models.py (90.0%): Missing lines 271,274,309
openedx/adg/lms/applications/test/factories.py (100%)
openedx/adg/lms/applications/views.py (100%)
-------------
Total:   196 lines
Missing: 4 lines
Coverage: 97%
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/946/workflows/workflows_id/jobs/946/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/course_meta/admin.py (100%)
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/management/commands/tests/test_update_is_prerequisite_courses_passed.py (100%)
openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py (97.5%): Missing lines 33
openedx/adg/lms/applications/models.py (90.0%): Missing lines 271,274,309
openedx/adg/lms/applications/test/factories.py (100%)
openedx/adg/lms/applications/views.py (100%)
-------------
Total:   195 lines
Missing: 4 lines
Coverage: 97%
-------------
```

**Comment by zahid-arbisoft** [comment]
> **pylint:** ❌
> **pycodestyle:** ✔️
> **iSort:** ✔️
> 
> ```
> 
> ```

@ahmed-faraz46 @asadali145 Pylint is failing without any reason and without a message, please ignore it. It needs to be fixed

**Review by asadali145 — DISMISSED** [review]
LGTM

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/955/workflows/workflows_id/jobs/955/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/course_meta/admin.py (100%)
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/management/commands/tests/test_update_is_prerequisite_courses_passed.py (100%)
openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py (97.5%): Missing lines 33
openedx/adg/lms/applications/models.py (86.7%): Missing lines 193,330,333,368
openedx/adg/lms/applications/test/factories.py (100%)
openedx/adg/lms/applications/views.py (100%)
-------------
Total:   195 lines
Missing: 5 lines
Coverage: 97%
-------------
```

**Comment by zahid-arbisoft** [comment]
## Diff Coverage Report - ADG
Check-out complete **[report](https://app.circleci.com/pipelines/github/OmnipreneurshipAcademy/edx-platform/956/workflows/workflows_id/jobs/956/artifacts)**. Coverage !!! :rocket:
```
-------------
Diff Coverage
Diff: origin/develop...HEAD, staged and unstaged changes
-------------
openedx/adg/common/course_meta/admin.py (100%)
openedx/adg/lms/applications/admin.py (100%)
openedx/adg/lms/applications/management/commands/tests/test_update_is_prerequisite_courses_passed.py (100%)
openedx/adg/lms/applications/management/commands/update_is_prerequisite_courses_passed.py (97.5%): Missing lines 33
openedx/adg/lms/applications/models.py (86.7%): Missing lines 193,330,333,368
openedx/adg/lms/applications/test/factories.py (100%)
openedx/adg/lms/applications/views.py (100%)
-------------
Total:   195 lines
Missing: 5 lines
Coverage: 97%
-------------
```

File: openedx/adg/lms/applications/models.py [hunk 1]
```diff
@@ -191,7 +190,9 @@ def prereq_course_scores(self):
         Returns:
             list: Prereq course name and score pairs
         """
-        prereq_course_overviews = CourseOverview.objects.filter(id__in=CourseMeta.prereq_course_ids.all())
+        prereq_course_overviews = CourseOverview.objects.filter(
```
Comment (ahmed-faraz46) [inline openedx/adg/lms/applications/models.py:193]: Good idea to display all courses in each group for now! In future probably we will be showing the ones user has enrolled in or the ones user has passed

**Review by ahmed-faraz46 — APPROVED** [review]
Reviewed last commit as rest of the PR was already reviewed in under this [old PR](We should return the). Looks good 👍

**Review by asadali145 — APPROVED** [review]
LGTM

**Pull request merged**